baseapp/baseapp.go

```diff
@@ -465,6 +465,8 @@ func (app *BaseApp) GetConsensusParams(ctx sdk.Context) *abci.ConsensusParams {
 		cp.Validator = &vp
 	}
 
+	cp.Version = &tmproto.VersionParams{AppVersion: app.appVersion}
```

Comment:
nice, so now we should just be able to call `app.SetVersion` instead of wrapping endblock, I can dig it